x-pack/plugins/enterprise_search/public/applications/app_search/components/create_engine/constants.ts

```diff
@@ -67,3 +67,70 @@ export const CREATE_ENGINE_SUCCESS_MESSAGE = i18n.translate(
     defaultMessage: 'Successfully created engine.',
   }
 );
+
+export const SUPPORTED_LANGUAGES = [
```

Comment:
[Discussion to use a local constant for these values](https://github.com/elastic/ent-search/pull/2990#issuecomment-770081169)